### PR TITLE
Bugfix/changes to task views

### PR DESCRIPTION
**Pull request by joefitter**

### CODE REVIEW

File: pages/task/read/content/base.js [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+    title: 'Current status',
+    'with-licensing': 'With licensing',
+    'referred-to-inspector': 'Referred to inspector',
+    'returned-to-applicant': 'Returned to applicant',
+    'withdrawn-by-applicant': 'Withdrawn by applicant',
+    'with-ntco': 'With NTCO',
+    'ntco-endorsed': 'NTCO endorsed',
+    'inspector-recommended': 'Inspector recommended for approval',
+    'inspector-rejected': 'Inspector recommended for rejected'
+  },
```
Comment (lennym) [inline pages/task/read/content/base.js]: I think this list now exists in a number of different places, and not necessarily in the same form. Can we consolidate?

Comment (joefitter) [reply pages/task/read/content/base.js]: yes, I did have a look in asl-constants but couldn't find it

Comment (joefitter) [reply pages/task/read/content/base.js]: I guess it just needs to be in the base content for all tasks, rather than asl-constants

Comment (lennym) [reply pages/task/read/content/base.js]: It's not in constants, I meant various content files in `pages/task/*`

Comment (lennym) [reply pages/task/read/content/base.js]: https://github.com/UKHomeOffice/asl-pages/blob/master/pages/task/read/content/success.js#L17-L41

Comment (lennym) [reply pages/task/read/content/base.js]: Also here: https://github.com/UKHomeOffice/asl-pages/blob/master/pages/task/list/content/index.js#L26-L36

File: pages/task/content/status.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  },
+  resolved: {
+    state: 'Resolved',
+    action: 'Update licence'
+  },
+  rejected: {
+    state: 'Rejected',
+    action: 'Reject'
+  }
+};
```
Comment (wheelsandcogs) [inline pages/task/content/status.js]: Like it 👍

Comment (lennym) [reply pages/task/content/status.js]: I agree with @wheelsandcogs

Comment (joefitter) [reply pages/task/content/status.js]: do either of you like it enough to merge it? :)

**Pull request merged**